storage/store.go

```diff
@@ -929,6 +929,11 @@ func (s *Store) Descriptor(nodeDesc *NodeDescriptor) (*StoreDescriptor, error) {
 // method, args & reply into a Raft Cmd struct and executes the
 // command using the fetched range.
 func (s *Store) ExecuteCmd(method string, args proto.Request, reply proto.Response) error {
+	select {
```

Comment:
I've fixed this issue by adding a LocalSender.RemoveStore method. Now ExecuteCmd isn't called after the store is stopped, so this check is gone.